lib/features/auto-resize/BpmnAutoResizeProvider.js

```diff
@@ -32,8 +32,11 @@ BpmnAutoResizeProvider.$inject = [
  * @return {boolean}
  */
 BpmnAutoResizeProvider.prototype.canResize = function(elements, target) {
-
-  if (!is(target, 'bpmn:Participant') && !is(target, 'bpmn:Lane') && !(is(target, 'bpmn:SubProcess'))) {
+  if (
```

Comment:
How is this related to the import? Isn't the scope of this pull request the import of diagrams? I assume we're going to tackle modeling in another pull request.

Reply:
That is correct, good catch. We should separate PRs to one topic, even when the "fix" to make modeling work is simple :+1: